.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "public/themes"]
 	path = public/themes
 	url = git://github.com/openSUSE/opensuse-themes.git
+
+[submodule "public/chameleon"]
+	path = public/chameleon
+	url = git@github.com:openSUSE-zh/chameleon.git
```

Comment:
Need someone who is the member of openSUSE organization to do this. I do not have that permission to transfer.